src/pages/studies/StudyTableContainer.tsx

```diff
@@ -32,24 +32,50 @@ type Props = StudiesResults & PropsFromRedux & { total: number };
 
 const StudyTable = (props: Props) => {
   const tableData = generateTableData(props);
+  const columns = studiesColumns(props.currentVirtualStudy, props.onClickStudyLink);
   const { total } = props;
 
+  const renderColumnSummary = (columns: any, data: any): Array<ReactNode> =>
+    columns.map((column: any) => {
+      if (column.children) {
+        return renderColumnSummary(column.children, data);
+      }
+
+      return (
+        <Table.Summary.Cell
+          key={column.dataIndex}
+          index={column.dataIndex}
+          className={styles.studyTableFooterCell}
+        >
+          <strong>{column.summary ? getColumnTotal(column.dataIndex, data) : ''}</strong>
+        </Table.Summary.Cell>
+      );
+    });
+
+  const getColumnTotal = (dataIndex: string, data: Array<StudiesResult>) => {
+    const result = data.map((study: StudiesResult | any) => study[dataIndex]);
```

Comment:
why `| any` ? 
data is supposed to be an array of StudiesResult isn't it ?

Reply:
It was to fix a typing error, but i found a better way.